### PR TITLE
chore: UnzipUtility constraints not needed

### LINKED ISSUE #20687: UnzipUtility constraints not needed
### Background

As the total size of the network software update files (entities 150..159) grows in size or contains individual files once uncompressed that exceed the limits defined in `UnzipUtility` (100MB per file, 1GB total), then the prepare upgrade unzip action will fail.

As there is implicit trust with the network upgrade file, it is not necessary to perform these checks and potentially be having to keep these constants in sync with the contents of the software update file.

Currently some cryptography jars can be large `202M Aug 18 15:38 hedera-cryptography-rpm-0.2.3-SNAPSHOT.jar`

```
2025-08-18 13:01:18.668 ERROR 277  ReadableFreezeUpgradeActions - Failed to unzip archive for NMT consumption
java.io.IOException: Zip bomb attack detected, aborting unzip!
	at com.hedera.node.app.service.networkadmin.impl.handlers.UnzipUtility.extractSingleFile(UnzipUtility.java:114)
	at com.hedera.node.app.service.networkadmin.impl.handlers.UnzipUtility.unzip(UnzipUtility.java:75)
	at com.hedera.node.app.service.networkadmin.impl.handlers.ReadableFreezeUpgradeActions.extractAndReplaceArtifacts(ReadableFreezeUpgradeActions.java:266)
	at com.hedera.node.app.service.networkadmin.impl.handlers.ReadableFreezeUpgradeActions.lambda$extractNow$0(ReadableFreezeUpgradeActions.java:229)
	at java.base/java.util.concurrent.CompletableFuture$AsyncRun.run(CompletableFuture.java:1804)
	at java.base/java.util.concurrent.CompletableFuture$AsyncRun.exec(CompletableFuture.java:1796)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:387)
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1312)
	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1843)
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1808)
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:188)
```

### Acceptance Criteria

- Remove restrictive logic in UnzipUtility

### Dependencies

_No response_

### Definition of Ready (DoR) Checklist

- [ ] Clear acceptance criteria
- [ ] Clear and detailed description
- [ ] Dependencies identified
- [ ] Links to documentation
- [ ] Should be completable in 2-3 Days
- [ ] Initial draft of Low-level design document
- [ ] At least high level test plan
- [ ] Groomed/Estimated

### Definition of Done (DoD) Checklist

- [ ] Acceptance Criteria complete
- [ ] No Codacy issues greater than minor (in new code)
- [ ] JavaDocs updated/created
- [ ] Code commented
- [ ] Unit tests created/updated
- [ ] 80% test code coverage (in new code)
- [ ] Happy Path and major negative cases in HAPI tests as applicable

### PR DESCRIPTION
**Pull request by derektriley**
**Description**:
This pull request removes zip bomb protection logic from the `UnzipUtility` class and updates the corresponding tests to reflect these changes. The zip bomb protection code, which previously limited the number and size of entries in zip files and total size, has been eliminated. The test suite has been refactored to remove tests for these protections, add new tests for directory-only and empty zip files, and improve coverage for error cases such as zip slip attacks and invalid parameters.

See #20687 for more information

### Removal of zip bomb protection logic

* Removed all threshold constants and checks in `UnzipUtility` that previously protected against zip bomb attacks by limiting the number of entries, total uncompressed size, individual entry size, and compression ratio. (`hedera-node/hedera-network-admin-service-impl/src/main/java/com/hedera/node/app/service/networkadmin/impl/handlers/UnzipUtility.java`)

### Refactoring and enhancement of tests

* Removed tests and code related to large zip files and zip bomb detection, and introduced new tests for directory-only and empty zip files, as well as improved error case coverage (e.g., invalid zips, zip slip, directory creation failures, and null parameters). (`hedera-node/hedera-network-admin-service-impl/src/test/java/com/hedera/node/app/service/networkadmin/impl/test/handlers/UnzipUtilityTest.java`)

**Related issue(s)**:

Fixes #20687

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

File: hedera-node/hedera-network-admin-service-impl/src/test/java/com/hedera/node/app/service/networkadmin/impl/test/handlers/UnzipUtilityTest.java [hunk 1]
```diff
@@ -85,33 +82,19 @@ void setup() throws IOException {
...
-        int targetSize = 104857600; // 100MB in bytes
-        byte[] data = new byte[targetSize + 1]; // Initialize byte array, size greater than threshold 100MB
-        SecureRandom random = new SecureRandom();
-        random.nextBytes(data); // Fill byte array with random bytes
-        return data;
+        // set up empty zip file
+        testZipEmpty = new File(zipSourceDir + "/testZipEmpty.zip");
+        try (ZipOutputStream out = new ZipOutputStream(new FileOutputStream(testZipEmpty))) {
+            // Create empty zip
+        }
```
Comment (mhess-swl) [inline hedera-node/hedera-network-admin-service-impl/src/test/java/com/hedera/node/app/service/networkadmin/impl/test/handlers/UnzipUtilityTest.java:97]: I don't see anything that cleans up the temporary dir after all the tests run. Would be nice, but not a blocker

Comment (derektriley) [reply hedera-node/hedera-network-admin-service-impl/src/test/java/com/hedera/node/app/service/networkadmin/impl/test/handlers/UnzipUtilityTest.java:97]: Should be fine as JUnit will attempt to cleanup the temporary directories
